lib/capybara/registrations/servers.rb

```diff
@@ -28,6 +28,7 @@
   options = default_options.merge(options)
 
   conf = Rack::Handler::Puma.config(app, options)
+  conf.options.finalize_values
```

Comment:
I think you can do this instead

```suggestion
  conf.clamp
```

See

* https://github.com/puma/puma/blob/v5.0.4/lib/puma/configuration.rb#L127-L130
* https://github.com/puma/puma/blob/v5.0.4/lib/puma/configuration.rb#L224-L228